frontend/Dockerfile

```
@@ -1,9 +1,22 @@
-FROM node:18-alpine
+# Builder image
+FROM node:18-alpine as builder
```

Don't we need docker-io here, too?

---

Yeah, that would make sense, thanks! 👍 

```suggestion
FROM docker.io/node:18-alpine as builder
```